### PR TITLE
Exercise1

### PR DESCRIPTION
**Pull request by stevelee2497**

### CODE REVIEW

File: Execise1/bai11/Program.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            Console.Write("enter a full name : ");
+            var name = Convert.ToString(Console.ReadLine());
+            var formatedName = FormatName(name);
+            Console.WriteLine("Output name : " + formatedName);
+            Console.ReadKey();
+        }
+
+        private static string FormatName(string name)
+        {
+            return "";
```
Comment (voquanghoa) [inline Execise1/bai11/Program.cs]: wtf

File: Execise1/bai8/Program.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+        private static bool isPrime(int number)
+        {
+            if (number < 4)
+            {
+                return true;
+            }
+            else
+            {
+                for (var i = 2; i * i <= number; i++)
```
Comment (voquanghoa) [inline Execise1/bai8/Program.cs]: Something went wrong

File: Execise1/lul/Program.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        static void Main(string[] args)
+        {
+            Console.Write("input a number : ");
+            int number_a = Convert.ToInt32(Console.Read());
+
+            Console.Write("input another number : ");
+            int number_b = Convert.ToInt32(Console.Read());
+
+            int result = gcd(number_a, number_b);
+            Console.WriteLine(result);
```
Comment (voquanghoa) [inline Execise1/lul/Program.cs]: Bad convention

File: Execise1/bai8/Program.cs (hunk 2 above)
Comment (stevelee2497) [reply Execise1/bai8/Program.cs]: What's the wrong sir ? 
the program is working fairly well.

Comment (voquanghoa) [reply Execise1/bai8/Program.cs]: Sorry my bad. I seem it was `i <= number` instead of `i * i <= number`

File: Execise1/bai11/Program.cs [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            var formatedName = FormatName(name);
+            Console.WriteLine("Output name : " + formatedName);
+            Console.ReadKey();
+        }
+
+        private static string FormatName(string name)
+        {
+            var firstPos = 0;
+            var lastPos = name.Length;
+            while (name[firstPos] == ' ') firstPos++;
```
Comment (voquanghoa) [inline Execise1/bai11/Program.cs]: C programming solution :D Use `name = name.Trimp()` instead

File: Execise1/bai11/Program.cs [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+            while (name[lastPos-1] == ' ') lastPos--;
+            StringBuilder formatedName = new StringBuilder();
+            var isFirstLetter = true;
+            for (int i = firstPos; i < lastPos; i++)
+            {
+                if (isFirstLetter)
+                {
+                    formatedName.Append(name[i] < 'a' ? name[i] : (char)(name[i] - 'a' + 'A'));
```
Comment (voquanghoa) [inline Execise1/bai11/Program.cs]: Replace `name[i] < 'a' ? name[i] : (char)(name[i] - 'a' + 'A')` by `name[i].ToUpper()`

File: Execise1/bai11/Program.cs [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+                    isFirstLetter = false;
+                }
+                else
+                {
+                    if (name[i] != ' ')
+                    {
+                        formatedName.Append(name[i] >= 'a' ? name[i] : (char) (name[i] + 'a' - 'A'));
```
Comment (voquanghoa) [inline Execise1/bai11/Program.cs]: Replace `name[i] >= 'a' ? name[i] : (char) (name[i] + 'a' - 'A')` by `name[i].ToLower()`

File: Execise1/bai11/Program.cs [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+                    }
+                    else
+                    {
+                        if (name[i + 1] != ' ')
+                        {
+                            formatedName.Append(name[i]);
```
Comment (voquanghoa) [inline Execise1/bai11/Program.cs]: Use `name = name.ReplaceAll("  ", " ")` to repace all double space by single space

File: Execise1/bai6/Program.cs [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+namespace bai6
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("enter three edges of a triangle : ");
+            Console.Write("edge a : ");
+            var edgeA = Convert.ToInt32(Console.ReadLine());
```
Comment (voquanghoa) [inline Execise1/bai6/Program.cs]: Make a common method and call `var edgeA = InputPromp("edge a : ");`

**Pull request merged**